app/controllers/airlines_controller.rb

```diff
@@ -6,7 +6,7 @@ def index
 
   def show
     @airline = Airline.find(params[:id])
-    @reviews = @airline.reviews
+    # @reviews = @airline.reviews
```

Comment:
It's bad practice to leave in commented-out code -- if this line isn't needed, please just delete it.